tests/test_fin_class.py

```diff
@@ -69,12 +69,12 @@ def test_set_all_configs():
   bn_file = "{0}/../utils/test_files/bn_configs_rows.txt".format(this_path)
   db = json.load(open(bn_file))
   #no db connection, set_all_configs returns False
-  assert (fin_worker.create_dumplist() == False)
+  assert (fin_worker._FinClass__create_dumplist() == False)
```

Comment:
@cderb  What is this doing ?

Reply:
@JehandadKhan I made create_dumplist a private function with __create_dumplist.
This mangles the name, so to access from outside the class _FinClass needs be prepended.